Client/Frontend/Browser/BrowserViewController/BrowserViewController+TabToolbarDelegate.swift

```diff
@@ -99,24 +99,23 @@ extension BrowserViewController: TabToolbarDelegate, PhotonActionSheetProtocol {
                 TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .logins)
             }
         }
-
-        let optionalActions = [syncAction, viewLogins].compactMap { $0 }
+        
+        var section1 = getLibraryActions(vcDelegate: self)
+        var section2 = getOtherPanelActions(vcDelegate: self)
+        let section3 = getSettingsAction(vcDelegate: self)
```

Comment:
nit: Any reason why section1 and section2 are vars and not let like sections3

Reply:
I am appending to them later